packages/livekit-server-sdk/src/index.ts

```diff
@@ -16,6 +16,8 @@ export {
   EncodingOptions,
   EncodingOptionsPreset,
   GCPUpload,
+  ImageCodec,
+  ImageFileSuffix,
```

Comment:
just double checking that these two exports are the only thing we need for enabling SRT output?

Reply:
these are unrelated to SRT. StreamProtocol should have SRT now though